### DISCUSSION TITLE
vercel couldn't detect mantine template

### DISCUSSION THREAD

**Discussion by 1rens1** [mantinedev/mantine#1451] [Q&A] [answered]
Hi, I recently created a NextJS app with the [Mantine Next Template](https://github.com/mantinedev/mantine-next-template). I wanted it to deploy to vercel, when I typed the repo name there's nothing. vercel didn't identify the repo as a NextJS project.

How do I make vercel know that my repo is a NextJS project?

**Comment by rtivital** [comment, marked as answer]
You better ask this question in Vercel support
